ocamlformat-help.txt

```diff
@@ -316,22 +327,67 @@ OPTIONS (CODE FORMATTING STYLE)
            nested pattern-matching under the encompassing pattern-matching.
            The default value is wrap.
 
+       --no-align-cases
+           Disable align-cases.
+
+       --no-align-constructors-decl
+           Disable align-constructors-decl.
+
+       --no-align-variants-decl
+           Disable align-variants-decl.
+
        --no-break-infix-before-func
-           Break infix operators whose right arguments are anonymous
-           functions specially: do not break after the operator so that the
-           first line of the function appears docked at the end of line after
-           the operator.
+           Disable break-infix-before-func.
+
+       --no-break-sequences
+           Disable break-sequences.
+
+       --no-disable
+           Disable disable.
+
+       --no-disambiguate-non-breaking-match
+           Disable disambiguate-non-breaking-match.
+
+       --no-dock-collection-brackets
```

Comment:
It would be nice to have `--(no-)dock-collection-brackets` (only 1 entry in the doc, but still 2 options), but judging from the Cmdliner API I'm not sure this line can be customized without changing the real name of the option

Reply:
I think that's not possible. I agree it does not look good